compiling-source/sass/components/_wordpress.scss

```diff
@@ -0,0 +1,72 @@
+@media only screen and (min-width: 1000px) {
+  .sidebar {
+    top: 1rem;
+    position: relative;
+  }
+}
+
+.post-meta {
+  padding: 20px;
+  font-size: .75em;
+  line-height: 20px;
+  color: #757575;
+}
+
+.meta-block {
+  float: left;
+  margin: 5px 20px 5px 0;
+  position: relative;
+}
+.meta-block a {
+  color: #757575;
+}
+.post-date {
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  padding: 0px;
+}
+
+.post-title {
+  letter-spacing: 1px;
+}
+
+.meta-bar {
+  padding-top:5px;
+  padding-left:40px;
+  background-color: #f8f8f8;
+  color: #757575 !important;
+  font-size: 1em;
+  line-height: 25px;
+  height:50px;
+}
+.page-title {
+  letter-spacing: 1px;
+  position: absolute;
+  margin: 7.5%;
+  margin-left: 3.5%;
+  color: #fff;
+}
+
+.read-more {
+  line-height: 16px;
+  display: block;
+  font-size: .75em;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  margin: 0;
+  color: #757575 !important;
+}
+.thumbnail-placeholder {
+  width: 100%;
+  height: 15vw;
+  background-color: #005e38;
```

Comment:
All hard-coded colors need to be changed to the colors defined in variables.

Reply:
This is the only occurence I could find, are there any others?